feat: add elements to treasure hunt

AkiPiy opened the pull request:
Elements are in the map.
Triggers are put.
Cleaning ok :)

Eggs : on map_sky
![image](https://user-images.githubusercontent.com/81775278/118029928-eb41ed80-b364-11eb-8e68-a35e61724345.png)

Salt : on map_market
![image](https://user-images.githubusercontent.com/81775278/118030012-0ca2d980-b365-11eb-8308-902109fb9e82.png)

Flour: on map.json
![image](https://user-images.githubusercontent.com/81775278/118030085-217f6d00-b365-11eb-8e60-ed49327b8f23.png)

Milk : on map.json
![image](https://user-images.githubusercontent.com/81775278/118030139-30661f80-b365-11eb-9c65-93e545120e1e.png)

Sugar (a little bit hidden) : on map-sponsors
![image](https://user-images.githubusercontent.com/81775278/118030251-54c1fc00-b365-11eb-889c-4186d024f76a.png)

Butter (hidden too) : on map-sponsors
![image](https://user-images.githubusercontent.com/81775278/118030310-686d6280-b365-11eb-892b-628f0b384959.png)

javamind commented:
Il y a un conflit sur la PR :disappointed: Est ce que tu peux rebaser ? 
Pour la carte map_market je croyais qu'on ne l'utilisait plus

AkiPiy commented:
Oui pas de souci j'essaye de la rebase, mon gitkraken rame pour le moment du coup je n'y arrive pas :( . Héhé j'ai pas dû tout suivre sur la conv, je vais planquer le sel ailleurs alors

AkiPiy committed:
feat: add treasure hunt

feat: add exit to go back to sand map


feat: add clouds to fix ground limits 


feat: add cotton candy


feat: add salt


feat: add butter


feat: add fmilk and sugar


feat: add sugar


fix: correct the size of element


feat: add trigger to get codename of treasure hunt elements


feat: add montgolfiere


feat: fix details


feat: fix details 2

al2b commented:
👏👏👏👏👏👏👏👏
